build.rs

```diff
@@ -126,6 +126,10 @@ fn cpu_supports_pt() -> bool {
 fn main() {
     let mut c_build = gcc::Build::new();
 
+    // We need the C_DEPS_PATH to be absolute so that our consumers inherit correct linker paths.
+    let mut c_deps_path_abs = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap());
+    c_deps_path_abs.push(C_DEPS_PATH);
```

Comment:
This doesn't feel right to me: `C_DEPS_PATH` should be `C_DEPS_LEAF` if it's relative to `CARGO_MANIFEST_DIR`?

Reply:
A path can be relative :P I've never heard of calling a directory a leaf.

If you feel strongly, how about instead `C_DEPS_DIR` instead?

Reply:
Works for me.